base/domain-afr.yml

```diff
@@ -0,0 +1,305 @@
+        *1.* JA
+        *2.* NEE
+
+  utter_ask_symptoms_weight:
+    - text: |
+        ⬛⬛⬛⬛⬜⬜
+
+        Verloor jy gewig sonder dat jy probeer?
+
+        Antwoord met
+        *1.* JA
+        *2.* NEE
+
+  utter_ask_exposure:
+    - text: |
+        ⬛⬛⬛⬛⬛⬜
+
+        Het jy 'n hoë risiko vir TB?
+
+        _Die kanse dat jy TB het is hoog indien:_
+        _Jy saam met iemand woon wat TB het/gehad het die afgelope jaar, OF as jy TB gehad die afgelope 2 jaar, OF as jy MIV-positief is._
+
+        Antwoord 1, 2 of 3
+        *1.* My risiko om TB te hê, is hoog
+        *2.* My risiko om TB te hê, is nie hoog nie
+        *3.* Ek weet nie of my risiko hoog is nie
+
+  utter_ask_tracing:
+    - text: |
+        ⬛⬛⬛⬛⬛⬛
+
+        Laastens, bevestig asseblief dat, sover jy weet, die inligting wat jy gegee het *korrek* is en dat jy die Nasionale Departement van Gesondheid toestemming gee om jou te kontak indien nodig?
+
+        Antwoord 1 of 2
+        *1.* JA
+        *2.* NEE
+
+  utter_risk_high:
+    - text: |
+        *Jou antwoorde op die vrae toon dat jy binne die volgende week vir 'n TB-toets moet gaan*
+
+        - Gaan na jou plaaslike kliniek vir 'n gratis TB-toets.
+        - Sit asseblief jou gesigmasker op voordat jy by die kliniek ingaan.
+
+  utter_risk_moderate:
+    - text: |
+        *Jou antwoorde op die vrae toon dat jy binne die volgende week vir 'n TB-toets moet gaan*
+
+        - Gaan na jou plaaslike kliniek vir 'n gratis TB-toets.
+        - Sit asseblief jou gesigmasker op voordat jy by die kliniek ingaan.
+
+  utter_risk_low:
+    - text: |
+        *Jou antwoorde op die vrae toon dat jy binne die volgende week vir 'n TB-toets moet gaan*
+
+        - Gaan na jou plaaslike kliniek vir 'n gratis TB-toets.
+        - Sit asseblief jou gesigmasker op voordat jy by die kliniek ingaan.
+
+  utter_risk_low_unknown_exposure:
+    - text: |
+        *Jy benodig nie tans 'n TB-toets nie.*
+
+        - Indien jy begin hoes, koors kry, gewig verloor of snags sweet, besoek jou naaste kliniek.
+
+  utter_follow_up_request:
+    - text: |
+        📲 Om opvolg-boodskappe te ontvang, antwoord *IN*
+      claim: release
+
+  utter_keywords:
+    - text: |
+        💊 Antwoord *GENESING* vir inligting oor behandeling
+        📗 Antwoord *BRONNE* vir inligting oor simptome en TB-toetsing
+        📌 Antwoord met *KIESLYS* om na die hoof-kieslys terug te keer"
+      claim: release
+
+  utter_opt_in_yes:
+    - text: |
+        Dankie dat jy gekies het om opvolg-boodskappe te ontvang.
+
+        📌 Antwoord met *KIESLYS* om na die hoof-kieslys terug te keer
+      claim: release
+
+  # TODO: add translation
+  utter_do_tb_screening:
+    - text: |
+        Please reply with *TB* to start your TB check.
+      claim: release
+
+  # claim: revert will ignore the message text, and reevaluate this message using
+  # turn automation
+  # TODO: add translation
+  utter_exit:
+    - text: "You can do your TB Check anytime by typing the word *TB*."
```

Comment:
This message text will be ignored (due to claim: revert), so we don't need a translation here